internal/app/commands/vacuum/vacuum.go

```diff
@@ -52,7 +52,7 @@ func createVacuumRequest(c *cli.Context) *http.Request {
 			"NodeBlobVacuumTask", "BinaryBlobVacuumTask", "SegmentVacuumTask", "VersionTableVacuumTask", "SnapshotsVacuumTask",
 		}
 	} else {
-		params["tasks"] = []string{"SegmentVacuumTask", "VersionTableVacuumTask", "SnapshotsVacuumTask"}
+		params["tasks"] = []string{"VersionTableVacuumTask", "SnapshotsVacuumTask"}
```

Comment:
Why is SegmentVacuumTask removed from params but is still present in docs ?
Same goes for BinaryBlobVacuumTask, NodeBlobVacuumTask.

Reply:
This task will be used only when `--blob, -b` are provided.